coffee-cdi/src/main/java/hu/icellmobilsoft/coffee/cdi/logger/LogContainer.java

```diff
@@ -279,17 +214,19 @@ private String format(String format, Object... arguments) {
     }
 
     private class Log {
+        private OffsetDateTime logDateTime;
         private LogLevel level;
         private String message;
 
         private Log(LogLevel level, String message) {
+            this.logDateTime = OffsetDateTime.now();
             this.level = level;
             this.message = message;
         }
 
         @Override
         public String toString() {
-            return level + ":" + message;
+            return MessageFormat.format("[{}]{}:{}",DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(logDateTime),level,message);
```

Comment:
parameter nelkul ez mukodik?

Reply:
Véletlen kimaradtak a számok, köszi!

Reply:
Legyszives teszteld le minden kodsort, mert kellemetlen lenne ha a projekteken kezdene elszalni

Reply:
Rendben, Írtam rá tesztet, hogy biztos ami biztos jó legyen, és így nemcsak letesztelem, de junit teszt is keletkezik.